File: reddio-csharp/reddio.api/V1/Rest/FeeInfo.cs
using Newtonsoft.Json;

namespace Reddio.Api.V1.Rest
{
    public class FeeInfo
    {
        public FeeInfo(long feeLimit, string tokenId, long sourceVaultId)
        {
            FeeLimit = feeLimit;
            TokenId = tokenId;
            SourceVaultId = sourceVaultId;
        }

        public FeeInfo()
        {
        }

        [JsonProperty("fee_limit")] public long FeeLimit;
        [JsonProperty("token_id")] public string TokenId;
        [JsonProperty("source_vault_id")] public long SourceVaultId;
    }
}

File: reddio-java/reddio-api/src/main/java/com/reddio/api/v1/rest/FeeInfo.java
package com.reddio.api.v1.rest;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor(staticName = "of")
public class FeeInfo {
    @JsonProperty("fee_limit")
    private Long feeLimit;

    @JsonProperty("token_id")
    private String tokenId;

    @JsonProperty("source_vault_id")
    private Long sourceVaultId;
}
